api/src/test/java/uk/gov/dhsc/htbhf/dwp/service/EligibilityServiceTest.java

```diff
@@ -33,18 +42,64 @@
     @MockBean
     private RestTemplate restTemplate;
 
+    @MockBean
+    private UCHouseholdRepository ucHouseholdRepository;
+
+    @MockBean
+    private LegacyHouseholdRepository legacyHouseholdRepository;
+
     @Autowired
     private EligibilityService eligibilityService;
 
     @Test
-    void shouldCreateEligibilityRequest() {
+    void shouldReturnResponseFromUCDatabaseIfFound() {
+        EligibilityRequest eligibilityRequest = anEligibilityRequest();
+        UCHousehold household = aHousehold();
+        given(ucHouseholdRepository.findHouseholdByAdultWithNino(anyString())).willReturn(Optional.of(household));
+
+        var response = eligibilityService.checkEligibility(eligibilityRequest);
```

Comment:
You should specify the response type rather than use `var` as per our discussion on this subject (plus in tests below too).

Reply:
Done.